plugins/processors/awsappsignals/internal/resolver/attributesresolver.go

```diff
@@ -62,11 +63,16 @@ func (r *attributesResolver) Stop(ctx context.Context) error {
 }
 
 type hostedInAttributeResolver struct {
+	name         string
 	attributeMap map[string]string
 }
 
-func newHostedInAttributeResolver(attributeMap map[string]string) *hostedInAttributeResolver {
+func newHostedInAttributeResolver(name string, attributeMap map[string]string) *hostedInAttributeResolver {
+	if name == "" {
+		name = "Generic"
```

Comment:
Could we instead use the PlatformGeneric const here as well instead of a string or do we want this to intentionally be `Generic` and not `generic`?

Reply:
We use `Generic` following the examples provided by CloudWatch metrics: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html.

Reply:
Added const `PlatformGeneric`.